src/default_panels/GraphSubplotsPanel.js

```diff
@@ -114,6 +114,33 @@ const GraphSubplotsPanel = (props, {localize: _}) => (
           {label: _('Sinusoidal'), value: 'sinusoidal'},
         ]}
       />
+      <Radio
+        label={_('Resolution')}
+        attr="resolution"
+        options={[
+          {label: _('1:110,000,000'), value: 110},
+          {label: _('1:50,000,000'), value: 50},
```

Comment:
that label title is a little weird no?

Reply:
that's what is in cloud - most importantly it will be translated to all languages :)